### PR TITLE
UI v2 new theme tooltip

### PR DESCRIPTION
**Pull request by harishshg**
Fixes # .

Changes proposed in this pull request:
- tooltip changes

@deepfence/engineering

### CODE REVIEW

**Comment by manV** [comment]
is this ready for review? @harishshg

File: deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -43,33 +45,37 @@ export const Tooltip = (props: TooltipProps) => {
               'radix-side-right:animate-slide-left-fade',
               'radix-side-bottom:animate-slide-up-fade',
               'radix-side-left:animate-slide-right-fade',
-              'inline-flex items-center rounded-lg px-3 py-2 shadow-sm',
-              'bg-gray-900 dark:bg-gray-700 max-w-xs',
+              'rounded-md pl-3 py-2 shadow-sm w-[200px]',
```
Comment (manV) [inline deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx:48]: let's not fix width, we can do max width.

File: deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -43,33 +45,37 @@ export const Tooltip = (props: TooltipProps) => {
               'radix-side-right:animate-slide-left-fade',
```
Comment (manV) [inline deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx:45]: `radix-` animations don't work anymore, use animations based on data attributes.

File: deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx [hunk 3]
```diff
@@ -43,33 +45,37 @@ export const Tooltip = (props: TooltipProps) => {
...
-              >
-                {content}
-              </span>
-            ) : (
-              content
-            )}
+            <>
+              {label && (
+                <span
+                  className={cx('text-p6', 'text-text-input-value', 'block', 'pb-[3px]')}
```
Comment (manV) [inline deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx:61]: please update your PR with latest changes in the base branch, `cx` and `twMerge` is replaced by new `cn` utility function you can import from the tailwind-preset package.

**Review by manV — CHANGES_REQUESTED** [review]

File: deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx (hunk 1 above)
Comment (harishshg) [reply deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx]: added max width

File: deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx (hunk 2 above)
Comment (harishshg) [reply deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx]: replaced with data attribute

File: deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx (hunk 3 above)
Comment (harishshg) [reply deepfence_frontend/packages/ui-components/src/components/tooltip/Tooltip.tsx]: replaced with cn utility function from tailwind-preset

**Pull request merged**